examples/Mailtrap.Example.Suppression/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Mailtrap;
+using Mailtrap.Accounts;
+using Mailtrap.Suppressions;
+using Mailtrap.Suppressions.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+
+HostApplicationBuilder hostBuilder = Host.CreateApplicationBuilder(args);
+
+IConfigurationSection config = hostBuilder.Configuration.GetSection("Mailtrap");
+
+hostBuilder.Services.AddMailtrapClient(config);
+
+using IHost host = hostBuilder.Build();
+
+ILogger<Program> logger = host.Services.GetRequiredService<ILogger<Program>>();
+IMailtrapClient mailtrapClient = host.Services.GetRequiredService<IMailtrapClient>();
+
+try
+{
+    var accountId = 12345;
+    IAccountResource accountResource = mailtrapClient.Account(accountId);
+
+    // Get resource for suppressions collection
+    ISuppressionCollectionResource suppressionsResource = accountResource.Suppressions();
+
+    var suppressionFilter = new SuppressionFilter
```

Comment:
I'd use a plural name

```suggestion
    var suppressionsFilter = new SuppressionsFilter
```